docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -159,7 +161,7 @@ sudo /usr/local/bin/besu/bin/besu --data-path=/var/lib/besu --data-storage-forma
 
 ### Cannot store generated private key
 
-`java.lang.IllegalArgumentException: Cannot store generated private key`
+- `java.lang.IllegalArgumentException: Cannot store generated private key`
```

Comment:
Recommend applying concatenate (we have mix of cannot and can't throughout these docs, probably need an issue just to slay)

Reply:
This is actually the error message in the terminal. It is taken from that message, so not a style guide edit ;/ but maybe we can raise it with the eng team.